airbyte-integrations/connectors/source-monday/components.py

```diff
@@ -39,8 +148,7 @@ def _ensure_type(self, t: Type, o: Any):
             raise TypeError(f"{type(o)} {o} is not of type {t}")
 
     def _get_schema_root_properties(self):
-        schema_loader = JsonFileSchemaLoader(config=self.config, parameters={"name": self.name})
-        schema = schema_loader.get_json_schema()["properties"]
+        schema = self.schema_loader.get_json_schema()[self.name]["properties"]
```

Comment:
as mentioned before, if we can add interpolation to load in just the schema of the current stream, then we wouldn't need to have `[self.name]`